src/v/cluster/cluster_discovery.h

```diff
@@ -81,7 +93,7 @@ class cluster_discovery {
     // On a seed server with no data on it (i.e. a fresh node), this sends
     // requests to all other seed servers to determine if there is a valid
     // assignment of node IDs for the seeds.
-    ss::future<model::node_id> determine_node_id();
+    ss::future<registration_result> determine_node_id();
```

Comment:
nit: would be worth to update method comment and name

Reply:
done in extra commit